solr/solr-ref-guide/modules/query-guide/pages/standard-query-parser.adoc

```diff
@@ -315,7 +315,7 @@ or
 
 === The Boolean Operator "+"
 
-The `+` symbol (also known as the "required" operator) requires that the term after the `+` symbol exist somewhere in a field in at least one document in order for the query to return a match.
+The `\+` symbol (also known as the "required" operator) requires that the term after the `+` symbol exist somewhere in a field in at least one document in order for the query to return a match.
```

Comment:
Thanks for spotting and sending PR to fix! Maybe one more here?
```suggestion
The `\+` symbol (also known as the "required" operator) requires that the term after the `\+` symbol exist somewhere in a field in at least one document in order for the query to return a match.
```

Reply:
I thought so too, but apparently because the sentence ends without another quoted `+` sign, ASCIIDoc recognizes the second one correctly. Escaping it would also display the escape character together with the `+` sign. Weird behavior if you ask me.